pwiz_tools/Shared/Lib/Microsoft.VC90.MFC/x86/Microsoft.VC90.MFC.manifest

```diff
@@ -1,13 +1,7 @@
-<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<!-- Copyright (c) Microsoft Corporation.  All rights reserved. -->
+﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <assembly xmlns="urn:schemas-microsoft-com:asm.v1" manifestVersion="1.0">
-    <noInheritable/>
-    <assemblyIdentity
-        type="win32"
-        name="Microsoft.VC90.MFC"
-        version="9.0.30729.1"
-        processorArchitecture="x86"
-        publicKeyToken="1fc8b3b9a1e18e3b"
-    />
-    <file name="mfc90u.dll" />
-</assembly>
+    <noInheritable></noInheritable>
+    <assemblyIdentity type="win32" name="Microsoft.VC90.MFC" version="9.0.30729.6161" processorArchitecture="x86" publicKeyToken="1fc8b3b9a1e18e3b"></assemblyIdentity>
+    <file name="mfc90.dll"/>

```

Comment:
CompassXtract needs mfc90.dll, MSFR needs mfc90u.dll. So far I still have Skyline only including mfc90u.dll, but I'll have to see whether this manifest causes an issue when one of its files is missing.